content/applications/marketing/events/event_essentials.rst

```diff
@@ -2,283 +2,384 @@
+showing the current :guilabel:`Month`, by default.
 
-.. image:: event_essentials/events-questions-tab.png
+.. image:: event_essentials/calendar-view.png
    :align: center
-   :alt: View of an event form, open the questions tab, and add a question.
+   :alt: The calendar view of the Odoo Events dashboard in the Odoo Events application.
 
-Log internal notes or add ticket instructions
----------------------------------------------
+To change the type of calendar, click the default :guilabel:`Month` drop-down menu in the upper-left
+corner to reveal a drop-down menu of other calendar view options. Those options are:
+:guilabel:`Day`, :guilabel:`Week`, :guilabel:`Month` (default), :guilabel:`Year`, and
+:guilabel:`Show weekends` (default).
 
-In the :guilabel:`Notes` tab, there is the option to add a :guilabel:`Note` and/or :guilabel:`Ticket
-instructions`.
+Navigate to past or future dates using the :guilabel:`<--` or :guilabel:`-->` buttons. Quickly
+return to the current date by clicking the :guilabel:`Today` button.
 
-Under :guilabel:`Note`, internal notes can be left (such as: to-do lists, contact information,
-etc.) for event staff to reference. In the :guilabel:`Ticket instructions` field, helpful
-information for staff and attendees can be shared (like, directions to the venue, opening/closing
-times, etc.)
+The right sidebar showcases a smaller version of the monthly calendar, along with
+:guilabel:`Responsible` and :guilabel:`Template` sections, each listing selectable responsibles and
+event templates in the database, respectively.
 
-.. tip::
-   Type `/` in either text field (:guilabel:`Note` or :guilabel:`Ticket instructions`) to reveal a
-   sub-menu of :guilabel:`Structure` options. These options provide various formatting options to
-   ensure vital internal information is organized for event staff to review.
+Clicking on a date on the calendar reveals a :guilabel:`New Event` pop-up window.
 
-.. image:: event_essentials/events-notes-tab.png
+.. image:: event_essentials/new-event-popup-window.png
    :align: center
-   :alt: View of the Notes tab in Odoo Events.
+   :alt: The New Event pop-up window in the Odoo Events application.
 
-Invite attendees to the event
-=============================
+In this pop-up window, enter a :guilabel:`Name` for the event to be scheduled for that desired date.
 
-To invite people to an event, click the :guilabel:`Invite` button located in the top left corner of
-the event form.
+Then, click either :guilabel:`Create` to create the event and add it to the calendar, or
+:guilabel:`Edit` to create the event, add it to the calendar, and edit its configurations right away.
 
-Inside the invite template form, are options to sent email or SMS invites. Each message can be fully
-customized, and recipients can be added.
+If the user no longer wishes to add an event on this date, either click :guilabel:`Cancel` or
+:guilabel:`X` to remove the :guilabel:`New Event` pop-up window.
```

Comment:
I think because I'd also have to use an (X) icon beside this an "emoji safety placeholder" -- I don't think the emoji is super necessary in this instance, but if I get the same note, I'll definitely make the change (and give you full credit again! 😆 )